webrender/src/device/gl.rs

```diff
@@ -736,6 +736,16 @@ impl Drop for SharedDepthTarget {
     }
 }
 
+/// Describes the support for the glTexStorage* family of functions.
+/// Various combinations of gl/gles version and extensions can mean either
+/// not supported, supported but not for BGRA8 format, or full support.
+#[derive(PartialEq, Debug)]
+enum TexStorageSupport {
+    None,
```

Comment:
Why is this here? You can just use `Option<TexStorageSupport>` elsewhere.

Reply:
I can't claim to be an expert on idiomatic rust, but my logic is that the level of support for glTexStorage is a tri-state: No support, support for non-bgra, and full support. In terms of how we use the variable, there is nothing special about the None case, nor in common about the two would-be Some cases. So using an Option feels less conceptually correct, and would require messier matching code, in my opinion.